person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require_relative 'Nameable'
+
```

Comment:
### OPTIONAL 

- [ ] Kindly import `require_relative 'capitalize_decorator'
require_relative 'trimmer_decorator'` so you can test the example below ⬇️ 

```
person = Person.new(22, 'maximilianus')
p person.correct_name
 capitalizedPerson = CapitalizeDecorator.new(person)
p capitalizedPerson.correct_name
 capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
p capitalizedTrimmedPerson.correct_name
```